code/modules/jobs/job_types/assistant/gimmick_assistants.dm

```diff
@@ -0,0 +1,165 @@
+// This file contains the louadouts for the assistant gimmicks station trait! When active, gives assistants random stupid gear
+
+/// Parent type of gimmick loadouts for assistants for the functional assistant station traits
+/datum/outfit/job/assistant/gimmick
+	name = "Gimmick Assistant"
+	/// The weight of the outfit to be picked
+	var/outfit_weight = 0
+
+
+/datum/outfit/job/assistant/gimmick/bee
+	name = "Gimmick Assistant - Bee"
+	suit = /obj/item/clothing/suit/hooded/bee_costume
+	l_pocket = /obj/item/coupon/bee
+
+	outfit_weight = 5
+
+/obj/item/coupon/bee
+	name = "coupon - 70% off Beekeeping Starter Crate"
+	desc = "BEEEES???? AT AN AFFORDAbLE PORICE?!!!" //wordcoders seething
+
+	discounted_pack = /datum/supply_pack/organic/hydroponics/beekeeping_fullkit
+	discount_pct_off = 0.7
+
+/datum/outfit/job/assistant/gimmick/chicken
+	name = "Gimmick Assistant - Chicken"
+	suit = /obj/item/clothing/suit/costume/chickensuit
+	head = /obj/item/clothing/head/costume/chicken
+
+	l_hand = /obj/item/storage/fancy/egg_box/fertile
+
+	outfit_weight = 3
+
+/datum/outfit/job/assistant/gimmick/cyborg
+	name = "Gimmick Assistant - Cardborg"
+	suit = /obj/item/clothing/suit/costume/cardborg
+	head = /obj/item/clothing/head/costume/cardborg
+
+	r_hand = /obj/item/weldingtool/largetank
+	l_hand = /obj/item/stack/cable_coil/five
+
+	uniform = /obj/item/clothing/under/color/black
+
+	outfit_weight = 3
+
+/datum/outfit/job/assistant/gimmick/cyborg/post_equip(mob/living/carbon/human/equipped, visualsOnly)
+	. = ..()
+	var/obj/item/organ/internal/tongue/robot/robotongue = new ()
+	robotongue.Insert(equipped, movement_flags = DELETE_IF_REPLACED)
+
+/datum/outfit/job/assistant/gimmick/skater
+	name = "Gimmick Assistant - Skater"
+	head = /obj/item/clothing/head/helmet/redtaghelm
+	suit = /obj/item/clothing/suit/redtag
+
+	l_hand = /obj/item/melee/skateboard
+
+	uniform = /obj/item/clothing/under/color/orange
```

Comment:
i think you want to override get_jumpsuit for outfits that do this?

Reply:
Yeah probably, was thinking about what to do with it. We have greysuits in config, and a station trait to color it sometimes